File: RestauranteApp/Services/Comanda/Models/ComandaCompletaModel.cs
using System;

namespace RestauranteApp.Services.Comanda.Models
{
    class ComandaCompletaModel
    {
        public int ComandaId { get; set; }
        public int MesaId { get; set; }
        public DateTime DataHoraEntrada { get; set; }
        public DateTime? DataHoraSaida { get; set; }
        public float Valor { get; set; }
        public bool Paga { get; set; }
        public int QuantidadeClientes { get; set; }
    }
}

File: frontend/src/app/pages/home/models/comanda-completa.model.ts
import { PedidoListaModel } from "src/app/shared/models/pedido-lista.model";

export interface ComandaCompletaModel {

    comandaId: number;
    mesaId: number;
    quantidadeClientes: number;
    valor: number;
    dataHoraEntrada: number;
    paga: boolean;
    pedidos: PedidoListaModel[];
}